packages/grafana-data/src/index.ts

```diff
@@ -48,3 +48,4 @@ export { getLinksSupplier } from './field/fieldOverrides';
 export { CircularVector } from './vector/CircularVector';
 export { vectorator } from './vector/FunctionalVector';
 export { ArrayVector } from './vector/ArrayVector';
+export * from './dataframe/CircularDataFrame';
```

Comment:
moving the export from dataframe/index.ts here somehow avoided a circular dependency with this error in tests:
```
TypeError: Class extends value undefined is not a constructor or null
```